templates/bundle/resources/usr/local/bin/generate_thread_dump.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+function check_usage {
+	DELAY=5
+	NUMBER_OF_THREAD_DUMPS=20
+	THREAD_DUMP_FOLDER="${LIFERAY_HOME}/data/thread_dumps"
+
+	while [ "${1}" != "" ]
```

Comment:
Let's remove this part, it's so much simpler if we don't need to have the -d -f, etc.

first parameter: number of thread dumps, second is seconds between third is the dir.

Reply:
I would like to keep this, as:

- It's already used in other scripts like the probe_thread_dump.sh.
- It's much easier to understand when reading the code.
- It's much easier to use, as a simple --help command can explain all the arguments.